openvpn-monitor.py

```diff
@@ -625,7 +625,10 @@ def print_html_header(self):
         if self.logo:
             output('<a href="#" class="pull-right"><img alt="Logo" ')
             output('style="max-height:46px; padding-top:3px;" ')
-            output('src="images/{0!s}"></a>'.format(self.logo))
+            if "http://" in self.logo or "https://" in self.logo:
```

Comment:
Could we use `startswith` instead of `in`? Otherwise, this looks good.

Reply:
updated!